lib/jsdom/events/EventException.js

```diff
@@ -0,0 +1,27 @@
+﻿"use strict";
+
+var inheritFrom = require("../utils").inheritFrom;
+
+function EventException() {
+  if (arguments.length > 0) {
+    this._code = arguments[0];
+  } else {
+    this._code = 0;
+  }
+  if (arguments.length > 1) {
+    this._message = arguments[1];
+  } else {
+    this._message = "Unspecified event type";
+  }
+  Error.call(this, this._message);
+  if (Error.captureStackTrace) {
+    Error.captureStackTrace(this, EventException);
+  }
+}
+
+inheritFrom(Error, EventException, {
+  UNSPECIFIED_EVENT_TYPE_ERR: 0,
+  get code() { return this._code; }
+});
+
+module.exports.EventException = EventException;
```

Comment:
I would rather stay with one class per file in general. If we need specific exceptions we can deal with those (and name them appropriately, e.g. `event-target.js` instead of `EventTarget.js`).